### PR TITLE
CLI-313: adding new test to properly handling negative case of invali…

### PR DESCRIPTION
**Pull request by hungklu**
…d argument input

### CODE REVIEW

File: src/main/java/org/apache/commons/cli/DefaultParser.java [hunk 1]
```diff
@@ -580,7 +580,7 @@ private boolean isArgument(final String token) {
      * Tests if the specified token is a Java-like property (-Dkey=value).
      */
     private boolean isJavaProperty(final String token) {
-        final String opt = token.substring(0, 1);
+        final String opt = token.length() > 0? token.substring(0, 1): null;
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/cli/DefaultParser.java:583]: Use String#isEmpty and flip the ternary. A space is missing around the "?". Add a test.

Comment (garydgregory) [reply src/main/java/org/apache/commons/cli/DefaultParser.java:583]: Ping?

Comment (hungklu) [reply src/main/java/org/apache/commons/cli/DefaultParser.java:583]: I'm starting to resume the work as I've been not feeling well.

Comment (hungklu) [reply src/main/java/org/apache/commons/cli/DefaultParser.java:583]: > Ping?

I'm starting to resume the work as I've been not feeling well.

File: src/test/java/org/apache/commons/cli/DefaultParserTest.java [hunk 2]
```diff
@@ -18,9 +18,11 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 package org.apache.commons.cli;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/cli/DefaultParserTest.java:21]: These imports are added but not used?

File: src/main/java/org/apache/commons/cli/DefaultParser.java (hunk 1 above)
Comment (garydgregory) [reply src/main/java/org/apache/commons/cli/DefaultParser.java:583]: Ping?

**Review by garydgregory — CHANGES_REQUESTED** [review]
Hello. Ping.

**Comment by centic9** [comment]
I think this is now superseeded by PR #186 and thus can be closed without merging.

**Comment by garydgregory** [comment]
@hungklu 
Please fix your build or close this PR.

**Comment by garydgregory** [comment]
Closing: No reply.
